java/src/main/java/ai/rapids/cudf/ColumnVector.java

```diff
@@ -257,6 +348,58 @@ public boolean hasNulls() {
   // DATA MOVEMENT
   /////////////////////////////////////////////////////////////////////////////
 
+  private final HostColumnVector.NestedHostColumnVector copyToHostNestedHelper(ColumnViewPointerAccess deviceCvPointer) {
```

Comment:
I think this can be static.

Reply:
Done and fixed nullCount in the process here.